HW_10/task_2.ts

```diff
@@ -0,0 +1,29 @@
+type TCallback<T, U> = (currentValue: T, index: number, arr: T[]) => U
+
+function map<T, U>(arr: T[], callback: TCallback<T, U>): U[] {
+    const result = arr.map(function(currentValue, index) {
+        return callback(currentValue, index, arr);

```

Comment:
```suggestion
    const result = arr.map(callback);
```
I was waiting for a for loop, cheater)